src/utils/applyContext.ts

```diff
@@ -36,7 +39,10 @@ export function applyContext<T extends object>(
     str.replace(reDelimiters, (_, path: string) => {
       const value = getValueFromPath(context, path);
       if (Array.isArray(value)) return `{${value}}`;
-      if (value instanceof Object) return 'undefined';
+      if (value instanceof Object) {
+        const json = value.toJSON?.();
```

Comment:
I'm seeing that the error is coming from this line because of the .

Reply:
Got it. Optional chaining is not supported in Node 13 and below. I will fix it.